cmake/Modules/CodeCoverage.cmake

```diff
@@ -104,6 +107,9 @@ mark_as_advanced(
     CMAKE_EXE_LINKER_FLAGS_COVERAGE
     CMAKE_SHARED_LINKER_FLAGS_COVERAGE )
 
+set(GCOV_TOOL "${CMAKE_CURRENT_LIST_DIR}/../../../../lib/code_coverage/llvm-gcov.sh")
```

Comment:
This is not pretty, but I'm not sure how would be better. Basing it off ROS_ROOT, which is along the lines of `/opt/ros/noetic/share/ros`?